src/app/shared/table/table-data/table-data.component.ts

```diff
@@ -45,6 +49,18 @@ export class TableDataComponent implements OnChanges {
     return this.config.format?.type;
   }
 
+  translateData(rawData: string): string {
+    let translatedData: string = '';
+    if (rawData) {
+      if (this.config.translationName) {
+        // @ts-ignore
+        translatedData = enums[this.config.translationName][rawData];
+        translatedData = this.translateService.instant(translatedData || '');
```

Comment:
i think that the default option should be the raw data if no translation is found ... not ''

Reply:
Resolved.